### ISSUE TITLE
Reaction picker is cropped on the small screen

### ISSUE THREAD

**Issue by Altonss** [signalapp/Signal-Android#11546]
Let's begin with a checklist: Replace the empty checkboxes [ ] below with checked ones [x] accordingly. -->

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
The reaction picker is cropped on my phone. I have a small screen in comparaison to most modern phones (5,9 cm wide) and am running Signal 5.20.4 from the PlayStore on Android 5.0.1.

### Steps to reproduce
- Open Signal app
- Enter a conversation
- React to a message
 
**Actual result:** The reaction picker is cropped, the left side of rounded bubble around the reaction picker is cropped.
**Expected result:** The reaction picker should be moved a little bit to the right, or its size should be reduced to fit the screen size. The bubble should not be cropped!

### Screenshots
![image](https://user-images.githubusercontent.com/66519591/129621340-7a8c5104-aae0-4a29-ae8e-dfabf2dc9bc9.png)

### Device info

**Device:** Samsung Galaxy Core Prime
**Android version:** 5.0.1
**Signal version:** 5.20.4

### Link to debug log
https://debuglogs.org/69b543f75e098fc5a61a23cfb98d2052b971b065254787082426378744f149b3

**Comment by Altonss** [comment]
This is still a issue on 5.25.7
Is there a plan to fix this issue or is this a nonfix?
ping @greyson-signal

**Comment by Altonss** [comment]
> Is this still relevant?

Yes this issue is still relevant and the problem is still hapenning on 5.28.10

**Comment by Altonss** [comment]
Yes

**Comment by Altonss** [comment]
This issue is still relevant!!

**Comment by alex-signal** [comment]
I've kicked this over to design to get some specs around small screens.
